xla/stream_executor/gpu/gpu_cudamallocasync_allocator.cc

```diff
@@ -38,12 +92,17 @@ limitations under the License.
 
 namespace stream_executor {
 
-#if GOOGLE_CUDA
-static std::string GetCudaErrorMessage(CUresult result) {
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
+static std::string GetCudaErrorMessage(gpu::GpuStatus result) {
   const char* error;
-  cuGetErrorString(result, &error);
   const char* name;
-  cuGetErrorName(result, &name);
+#if GOOGLE_CUDA
+  GpuGetErrorString(result, &error);
+  GpuGetErrorName(result, &name);
+#elif TENSORFLOW_USE_ROCM
+  error = GpuGetErrorString(result);
+  name = GpuGetErrorName(result);
+#endif
```

Comment:
Maybe we can just use cuGetErrorString/cuGetErrorName and hipGetErrorString/hipGetErrorName there ? Because the function prototypes differ anyway, hence there is no much sense in defining generic versions GpuGetErrorString/GpuGetErrorName above

Reply:
Make sense.